src/pytest_check/plugin.py

```diff
@@ -34,6 +35,11 @@ def pytest_runtest_makereport(item, call):
             else:
                 report.longrepr = "\n".join(longrepr)
             report.outcome = "failed"
+            try:
+                raise AssertionError(report.longrepr)
+            except AssertionError:
```

Comment:
Maybe we need to raise some custom `pytest-check` exception

Reply:
Is this a question or a statement? I don't see a need for a custom pytest-check exception.
Let me know if that would help.